sdcm/nemesis.py

```diff
@@ -656,7 +671,45 @@ def disrupt_truncate(self):
         # do the actual truncation
         self.target_node.run_cqlsh(cmd='TRUNCATE {}.{}'.format(keyspace_truncate, table), timeout=120)
 
-    def _modify_table_property(self, name, val, filter_out_table_with_counter=False):
+    def _get_compaction_strategy(self, node, keyspace, table):
+        """Get a given table compaction strategy
+
+        Arguments:
+            node {str} -- ip of db_node
+            keyspace
+            table
+        """
+        list_tables_compaction = node.run_cqlsh('SELECT keyspace_name, table_name, compaction FROM system_schema.tables',
+                                                split=True)
+        compaction = 'N/A'
+        for row in list_tables_compaction:
+            if '|' not in row:
+                continue
+            list_stripped_values = [val.strip() for val in row.split('|')]
+            if list_stripped_values[0] == keyspace and list_stripped_values[1] == table:
+                compaction = CompactionStrategy.from_str(output_str=list_stripped_values[2].split("'")[3])
+        self.log.debug("Query result for {}.{} compaction is: {}".format(keyspace, table, compaction.value))
+        return compaction
+
+    def _get_non_system_ks_cf_list(self, node):
```

Comment:
I quite sure we already have this function somewhere in the code

Reply:
see above

Reply:
done